From 834d8164d8cc460331bf7d0ca3d0233e609a3d59 Mon Sep 17 00:00:00 2001
From: jinghua <jinghua@marvell.com>
Date: Mon, 2 Nov 2015 13:47:47 +0800
Subject: [PATCH 0462/1240] fix: armada3700: increase u-boot
 CONFIG_SYS_MALLOC_LEN to 5MB

    increase u-boot CONFIG_SYS_MALLOC_LEN to 5MB, which is aligned
    with previous armada SoCs in u-boot 2013.

    There is 1MB for neta driver to allocate RX buffer, another 1MB
    for eMMC driver, the rest will be used in future for other units.

Change-Id: I0ff3b04e51548b0e007d4bded6b734c293810e0e
Signed-off-by: jinghua <jinghua@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/24513
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 include/configs/mvebu-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/mvebu-common.h b/include/configs/mvebu-common.h
index 0281b0d..1dc389b 100644
--- a/include/configs/mvebu-common.h
+++ b/include/configs/mvebu-common.h
@@ -44,7 +44,7 @@
 #define CONFIG_SYS_SDRAM_BASE           0x00000000
 #define CONFIG_SYS_RESET_ADDRESS        0xffff0000
 #define CONFIG_SYS_MALLOC_BASE          (CONFIG_SYS_TEXT_BASE + (3 << 20))  /* TEXT_BASE + 3M */
-#define CONFIG_SYS_MALLOC_LEN           (1 << 20)    /* Reserve 1MB for malloc*/
+#define CONFIG_SYS_MALLOC_LEN           (5 << 20)	/* Reserve 5MB for malloc*/
 #define CONFIG_NR_DRAM_BANKS		(2)
 
 /* maybe need to set back to 0x7fff0 */
-- 
1.9.1

